fido-key-manager/src/main.rs

```diff
@@ -774,5 +777,20 @@ async fn main() {
             println!("YubiKey config:");
             println!("{cfg}")
         }
+
+        #[cfg(feature = "usb")]
+        Opt::Wink => {
+            use webauthn_authenticator_rs::usb::USBTransport;
+
+            println!("Insert a USB device...");
+            let transport = USBTransport::new().await.unwrap();
+            let mut stream = transport.watch().await.unwrap();
+            while let Some(event) = stream.next().await {
+                if let TokenEvent::Added(mut token) = event {
+                    token.wink().await.expect("failed to wink USB device");
```

Comment:
Instead of an expect, how 'bout we do
```suggestion
                    return token.wink().await.map_err(|err| eprintln!"failed to wink USB device: {:?}", err);
```

Reply:
This doesn't work, as the containing function (i.e. `main`) does not return a `Result`. The existing code nearby here is using expects/unwraps to exit the CLI with an error code.
I guess it makes sense to rework error-handling for all of `main` in the longer term.

Reply:
Ah, derp, true. Sigh.